plugins/chat/spec/requests/chat/api/direct_messages_controller_spec.rb

```diff
@@ -78,13 +78,25 @@ def create_dm_channel(user_ids)
         )
       end
 
-      it "createsa new dm channel" do
+      it "creates a new dm channel" do
         create_dm_channel(direct_message_user_ids)
 
         expect {
           post "/chat/api/direct-message-channels.json", params: { target_usernames: [usernames] }
         }.to change { Chat::DirectMessage.count }.by(1)
       end
+
+      it "returns existing dm channel when using chat link" do
```

Comment:
I think it should be more ` it "returns existing dm channel when upsert is true" do`

Reply:
Thanks, I missed that one 👍